Update theatre programming section to remove code block formatting (#155)
AvolitesLtd/TitanManual: docs/cue-lists/theatre-programming.md

> touchscreen. But fear not because you can program a theatre show on
> Titan using the keypad and this is how you do it.

Reviewer: ```suggestion
touchscreen. Theatre-style shows are still achievable within Titan
and this page covers the keypad syntax.
```

> - Labelled buttons on the console are shown like this: \<Enter\>. 

Reviewer: ```suggestion
- Labelled buttons on the console are shown like this: \<Enter\>
```

Author: Well this is a totally minor thing and I don't really mind either way but I would have thought this should have a full stop since all the other bullet points have a full stop?

## Setting Dimmer/Fixture Levels

>  Keypresses | Operation

Reviewer: ```suggestion
 Keypresses | Action
```
I think Action is used elsewhere in the manual and potentially fits better

------------|-------------------

>  n \<@\> \<@\> | Set fixture **n** to 100%

Reviewer: ```suggestion
 **n** \<@\> \<@\> | Set fixture **n** to 100%
```
These could be clearer if the **n** in the keypresses section matched the action section
